.changeset/odd-icons-travel.md

```diff
@@ -0,0 +1,13 @@
+---
+"@ima/plugin-testing-integration": minor
+"@ima/plugin-analytic-fb-pixel": minor
+"@ima/cli-plugin-scramble-css": minor
+"@ima/plugin-analytic-google": minor
+"@ima/cli-plugin-legacy-css": minor
+"@ima/cli-plugin-analyze": minor
+"@ima/plugin-websocket": minor
+"@ima/plugin-analytic": minor
+"@ima/plugin-atoms": minor
+---
```

Comment:
Only a subset of dependencies is listed here. Probably only the ones with changed files. But the file changes are mostly about adding / removing a comment, or re-ordering imports (but only in test files, which do not affect the published package), or prettier changes (only in plugin-atoms). These changes are definitely not minor.

Unless we want to publish all packages, because of the updated build dependencies in which case we should list probably all packages in this monorepo.

Reply:
Changeset updated